lib/charms/vault_k8s/v0/juju_facade.py

```diff
@@ -330,7 +331,7 @@ def grant_secret(
         secret.grant(relation)
 
     # Relation related methods
-    def get_relation_by_id(self, relation_name: str, relation_id: int) -> Relation:
+    def get_relation_by_id(self, name: str, id: int) -> Relation:
```

Comment:
The method name does not match its content, if we get it by id, why ask for the name? I'd change to `get_relation`, just like we have `get_relations` below (and not `get_relations_by_id`)

Reply:
✅